staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/envelope_test.go

```diff
@@ -230,10 +218,7 @@ func benchmarkRead(b *testing.B, transformer value.Transformer, valueLength int)
 // remove after 1.13
 func TestBackwardsCompatibility(t *testing.T) {
```

Comment:
I think this comment is wrong because we do not run storage migration automatically so we cannot assume that folks are not using the old non-cryptobyte format.

xref: https://github.com/kubernetes/kubernetes/pull/69249#discussion_r222387758